doc/python_api/rst/bge_types/bge.types.KX_Scene.rst

```diff
@@ -143,10 +143,15 @@ base class --- :class:`PyObjectPlus`
 
       :type: Vector((gx, gy, gz))
 
-   .. attribute:: lodGlobalScale
+   .. method:: setLodScale(pythonObjectList, scale)
```

Comment:
Because they did not want to iterate in a list with python. What I can understand. Here we iterate but in c++